### DISCUSSION TITLE
Is there a way to use decibels with regular units?

### DISCUSSION THREAD

**Discussion by endolith** [Qalculate/libqalculate#384] [Q&A]
For example, if I have an amplifier with a gain of (1+1000/200) V/V, this is also a gain of +15.56 dB.

Is there a way to calculate the dB value from the voltages in Qalculate?  

In Maxima I have a convenience function defined:

    db(x)  := float(20 * log(x) / log(10));

So I can type `dB(1+1000/200)` and it will return `15.563`.  Can I do the same in Qalculate?

**Comment by VK2BEA** [comment]
I could create a derived unit dBV to convert to and from dBV.
Look at how the dBm and dBW is defined and copy what is done there...
 
```
> (1 + (1000 / 200))V to dBV

  (1 + (1000 / 200)) × volt ≈ 15.56302501 dBV

> 10V to dBV

  10 × volt = 20 dBV

> 6dBV + 1V =

  warning: Log-based units were converted before calculation.
  (6 × dBV) + (1 × volt) ≈ 2.995262315 V

> 3dBW + 1W =

  warning: Log-based units were converted before calculation.
  (3 × dBW) + (1 × watt) ≈ 2.995262315 W

> 
```
